content/reference/endpoints/hosts.md

```diff
@@ -3,6 +3,10 @@
 1. TOC
 {:toc}
 
+## Overview
+
+This resource represents Hosts which are the representations of Users of the particular [Account](/reference/endpoints/accounts/) that can be used in the Inbox context to create [Inbox Participants](reference/endpoints/inbox_participants/).
```

Comment:
`This resource represents Hosts which are the representations of [...]` "represent" duplication makes this sentence read a bit weirdly, maybe  `This resource represents Hosts which are related to Users [...]` (or maybe you have some better ideas) ?

Reply:
I would keep it as it is correct and that word is better than "related", it's not a language contest for the most sophisticated vocabulary, so simplicity wins